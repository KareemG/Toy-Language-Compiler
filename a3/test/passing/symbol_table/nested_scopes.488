
{
    var a : integer

    {
        var a : integer
        var b : integer

        {
            var a : integer
            var b : integer
            var c : integer

            a := 0
            b := 1
            c := a + b
        }
    }
}
