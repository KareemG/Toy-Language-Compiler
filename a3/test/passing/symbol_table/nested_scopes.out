{
 Symbols:
 a: integer
 {
 Symbols:
 b: integer
 a: integer
 {
 Symbols:
 b: integer
 a: integer
 c: integer
 }
 }
